package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aws-actions-amazon-ecs-render-task-definition",
-  "version": "1.6.0",
+  "version": "1.6.1",
```

Comment:
why are we manually updating our release version?

Reply:
+1

Reply:
Ack, will remove manual update of release version